### DISCUSSION TITLE
GET /user/{user}/delete within Resources Controllers

### DISCUSSION THREAD

**Discussion by khalyomede** [laravel/framework#39426] [Ideas]
I personally often need a /delete route to display a confirm page to users. 

I usually like this pattern because the user is confident clicking a delete button will not immediately delete the resource (for example when the button is located in a table, where a miss click can happen).

Currently `Route::resource()` will not generate the `GET /user/{user}/delete` route, and I was curious if other folks would find it useful or not?

**Comment by vuvanly** [comment]
I think a confirmation dialog would solve your problem.

**Reply by khalyomede** [reply]
Not an ideal solution for me that want to load the least JS, but I agree it works!
